test/import.jl

```diff
@@ -1,7 +1,46 @@
 using OndaEDF: validate_arrow_prefix
 
+function test_preprocessor(l)
+    l = replace(l, ':' => '-')
+    l = replace(l, "\xf6"[1] => 'o') # remove umlaut (German)
+    l = replace(l, '\u00F3' => 'o') # remove accute accent (Spanish)
+    l = replace(l, '\u00D3' => 'O') # remove accute accent (Spanish)
+    # "EOG - L" and "EOG - R" should not be parsed as channel \minus channel
+    m = match(r"^\s*EOG[\s\-]+(?<lr>[LR])\s*"i, l)
+    if !isnothing(m)
+        l = "EOG$(m[:lr])"
+    end
+    # "L - EOG" and "R- EOG" should not be parsed as channel \minus channel
+    m = match(r"^\s*(?<lr>[LR])[\s\-]+EOG\s*"i, l)
+    if !isnothing(m)
+        l = "EOG$(m[:lr])"
+    end
+    # "C2M1" => "C2-M1" etc
+    m = match(r"\s*(?<channel>[fco][1234])\s*(?<ref>[am][12])\s*"i, l)
+    isnothing(m) && return l
+    return "$(m[:channel])-$(m[:ref])"
+end
+
+custom_extractors = [edf -> extract_channels_by_label(edf, signal_names, channel_names; preprocess_labels=test_preprocessor)
+                     for (signal_names, channel_names) in OndaEDF.STANDARD_LABELS]
+
```

Comment:
For tests, use `preprocess_labels` appropriate for `test_edf_to_samples_info.out`.